front/src/reducers/GlobalMap/index.ts

```diff
@@ -0,0 +1,108 @@
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { useDispatch, useSelector } from 'react-redux';
+
+import type { InfraErrorTypeLabel } from 'common/api/osrdEditoastApi';
+import { useOsrdContext } from 'common/osrdContext';
+import type { RootState } from 'reducers';
+import type { LayersSettings, MapSearchMarker, Viewport } from 'reducers/map';
+
+export const defaultMapSettings: MapSettings = {
+  mapStyle: 'normal',
+  showIGNBDORTHO: false,
+  showIGNSCAN25: false,
+  showIGNCadastre: false,
+  showOSM: true,
+  showOSM3dBuildings: false,
+  showOSMtracksections: false,
+  terrain3DExaggeration: 0,
+  smoothTravel: false,
+  layersSettings: {
+    buffer_stops: false,
+    electrifications: false,
+    neutral_sections: false,
+    detectors: false,
+    operational_points: true,
+    routes: false,
+    signals: false,
+    sncf_psl: false,
+    speed_limits: false,
+    speedlimittag: null,
+    switches: false,
+    tvds: false,
+    platforms: false,
+  },
+  viewport: {
+    latitude: 48.32,
+    longitude: 2.44,
+    zoom: 6.2,
+    bearing: 0,
+    pitch: 0,
+    padding: { top: 0, left: 0, bottom: 0, right: 0 },
+    width: 0,
+    height: 0,
+  },
+  mapSearchMarker: undefined,
+  lineSearchCode: undefined,
+};
+
+export default defaultMapSettings;
+
+export interface MapSettings {
```

Comment:
We prefer type to interface in the code base
```suggestion
export type MapSettings {
```